tokens.json

```diff
@@ -1429,5 +1429,13 @@
         "symbol": "USDT",
         "account": "issue.newdex",
         "chain": "eos"
+    },
+    {
+        "name": "STLCXXI",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/stlcxxi.png",
```

Comment:
you're missing your logo

Please provide the png

Reply:
"Uploads are disabled. File uploads require push access to this repository."

Reply:
I can't add logo